### DISCUSSION TITLE
Nessun intervento finora nel registro

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#587] [Richieste di aiuto]
Buonasera ,come mai non ci sono più interventi nel registro elettronico?

**Comment by trinko** [comment]
Lo sviluppo è atttualmente su un ramo separato (devel) in modo che le modifiche non possano essere scaricate con gli aggiornamenti.
Si tratta di modifiche che principalmente riguardano la gestione di gruppi all'interno della classe (es. classi articolate): questo ha richiesto modifiche importanti sul codice che, sino a quando non sono ben consolidate non è opportuno introdurre nel codice attualmente stabile.

Appena arriverò ad una versione definitiva, riporterò tutto nella futura versione 1.5.6.

**Comment by sempliciniluigi89** [comment]
Ho visto che in giro vi è un sito di giuaschool in costruzione al seguente
link :https://registrogiua.altervista.org

Volevo sapere se riguardasse sempre voi o una scuola privata

Il giorno sab 18 nov 2023 alle ore 00:00 Antonello Dessì <
***@***.***> ha scritto:

**Comment by trinko** [comment]
> Ho visto che in giro vi è un sito di giuaschool in costruzione al seguente link :https://registrogiua.altervista.org Volevo sapere se riguardasse sempre voi o una scuola privata 

Non è gestito da noi. Non so di cosa si tratti.

**Comment by trinko** [comment]
Penso di riuscire a rilasciare la nuova versione a dicembre.

Le principali modifiche nella nuova versione sono:
- gestione delle classi articolate (gruppi interni alla classe)
- gestione dei gruppi di religione (gruppo religione, gruppo materia alternativa, gruppo che non si avvale)
- gestione moduli per le prove di evacuazione
- visualizzazione informazioni alunni trasferiti su pagine del coordinatore e staff
- visualizzazione dettagli della lettura delle circolari per lo staff
- visualizzazione composizione del Consiglio di Classe per lo staff

**Comment by trinko** [comment]
No, mi spiace, per questo rilascio non è possibile.

**Comment by trinko** [comment]
No, ancora no.
